src/components/EditableParagraph.jsx

```diff
@@ -10,6 +10,7 @@ const EditablePagrapraph = (props) => {
 
   const paragraphEditor = useRef();
 
+  const [text, setText] = useState(value);
```

Comment:
What are the changes done here for? They don't seem related to the ticket

Reply:
Ah you added unit tests so codecov doesn't cry I think, but the refactoring here was to make this respond better or something?